scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -229,6 +230,67 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
   </div>
 {% endmacro %}
 
+{% macro group_panel() %}
+  <div class="card panel-default">
+    <div data-toggle='tooltip' class="panel-heading" title=""><span class="fa fa-users"></span>&nbsp;Group connected cases</div>
+    <div class="card-body">
+      {% if case_groups %}
+        {% for group_id, case_group in case_groups.items() %}
+          {{ modal_add_to_group(group_id) }}
+          <div class="row">
+            <div class="col">
+              <span class="fa fa-users"></span>&nbsp;
+            </div>
+            <div class="col text-right">
+              <div class="btn-group">
+                <button type="button" class="btn btn-outline-secondary form-control" data-toggle="modal" data-target="#add-case-group-{{group_id}}">
+                  <span class="fa fa-plus-square-o"></span>&nbsp;Add to group
+                </button>
+              </div>
+            </div>
+          </div>
+          <ul class="list-group">
+          {% for grouped_case in case_group %}
+            <li class="list-group-item">
+                <a href="{{ url_for('cases.case', institute_id=grouped_case.owner, case_name=grouped_case.display_name) }}"">{{ grouped_case.display_name }}</a>
+                <span class="pull-right">
+                  <a href="{{ url_for('cases.remove_case_group', institute_id=institute._id, case_name=grouped_case.display_name, case_group=group_id) }}" class="btn btn-link btn-sm">
+                    <span class="fa fa-remove"></span></a>
+                </span>
+            </li>
+          {% endfor %}
+          </ul>
+        {% endfor %}
+      {% endif %}
+      <a href="{{ url_for('cases.add_case_group',institute_id=institute._id, case_name=case.display_name) }}" class="btn btn-primary btn-xs text-white"><span class="fa fa-user-plus text-white mr-3"></span>Add group</a>
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro modal_add_to_group(group_id) %}
+  <form action="{{ url_for('cases.add_case_group', institute_id=institute._id, case_name="unused") }}" method="POST">
+    <div class="modal fade" id="add-case-group-{{group_id}}" role="dialog">
+      <input type="hidden" name="group_id" value="{{ group_id }}">
+      <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" id="case_pick_dialog">Add case to existing case group - case {{case.display_name}}</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <input type="text" name="other_case_name" class="typeahead_cases form-control" data-provide="typeahead" autocomplete="off" placeholder="Search..." cols="30" rows="10">Case name</input>
```

Comment:
Perhaps the placeholder for the input could be "search case name", so you can remove Case name under it

![image](https://user-images.githubusercontent.com/28093618/105152542-a1a86b00-5b07-11eb-8399-0ea8255f2a8d.png)